NewTutorials/Podfile

```diff
@@ -2,10 +2,10 @@ source "https://github.com/vidible/OneMobileSDK-releases.git"
 
 target 'Tutorial' do
 	platform :ios, 8.0
-  	pod 'OneMobileSDK-Xcode9.2'
+  	pod 'OneMobileSDK-Xcode9.2', '~> 2.20.2141'
```

Comment:
Why we need to specify this version? `pod 'OneMobileSDK-Xcode9.2'` will do the same.